<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminwindow</class>
 <widget class="QMainWindow" name="adminwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>281</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Admin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="vardas">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="pavarde">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>1301</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Studentų grupės</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="dalykai">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Dalykai</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="naudotojai">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Naudotojai</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="programos">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>Programos</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>410</y>
      <width>1301</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>730</y>
      <width>331</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QStackedWidget" name="widgets">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>740</y>
      <width>1311</width>
      <height>80</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QWidget" name="addnaudotojai" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-10</y>
        <width>1301</width>
        <height>80</height>
       </rect>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>10</y>
         <width>1301</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="addnaudotojas">
        <item>
         <widget class="QLineEdit" name="addid">
          <property name="placeholderText">
           <string>id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addvardas">
          <property name="placeholderText">
           <string>vardas</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addpavarde">
          <property name="placeholderText">
           <string>pavarde</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addtel">
          <property name="placeholderText">
           <string>tel nr</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addemail">
          <property name="placeholderText">
           <string>email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addkodas">
          <property name="placeholderText">
           <string>asmens kodas</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addtipas">
          <property name="placeholderText">
           <string>tipas</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pasalinti">
          <property name="text">
           <string>pašalinti</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="prideti">
          <property name="text">
           <string>pridėti</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1301</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="adddalykai_2">
       <item>
        <widget class="QLineEdit" name="addid_3">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="dalykaslist"/>
       </item>
       <item>
        <widget class="QComboBox" name="programoslist">
         <property name="maximumSize">
          <size>
           <width>60</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="semestras">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>semestras</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="mokytojaslist"/>
       </item>
       <item>
        <widget class="QPushButton" name="pasalinti_3">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>pašalinti</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="prideti_3">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>pridėti</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1301</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="adddalykai">
       <item>
        <widget class="QLineEdit" name="addid_2">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="adddalykas">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>dalyko pavadinimas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addaprasas">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>aprašas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pasalinti_2">
         <property name="text">
          <string>pašalinti</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="prideti_2">
         <property name="text">
          <string>pridėti</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="horizontalLayoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>1301</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="adddalykai_3">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Order by ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Order by GROUP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="addid_4">
         <property name="maximumSize">
          <size>
           <width>70</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string>id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="studentaslist"/>
       </item>
       <item>
        <widget class="QComboBox" name="grupelist"/>
       </item>
       <item>
        <widget class="QPushButton" name="pasalinti_4">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>pašalinti</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="prideti_4">
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>pridėti</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1790</x>
      <y>0</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/Screenshot_11.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1791</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/Screenshot_13.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1820</x>
      <y>90</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Išjungti</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
